Tools/LyTestTools/ly_test_tools/environment/file_system.py

```diff
@@ -142,7 +142,7 @@ def unzip(dest, src, force=False, allow_exists=False):
         if secs == 0:
             secs = 0.01
 
-        logger.info(
```

Comment:
All of the changes in this file should remain info as they are taking destructive action.

However I don't think any of these are currently called by current tests.